admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -82,12 +82,15 @@ class DataLoader(object):
     def __init__(self, configFiles, czarWmgr, workerWmgrMap={}, chunksDir="./loader_chunks",
                  chunkPrefix='chunk', keepChunks=False, skipPart=False, oneTable=False,
                  css=None, cssClear=False, indexDb='qservMeta', tmpDir=None,
-                 emptyChunks=None, deleteTables=False, loggerName=None, cssDb='qservCssData'):
+                 emptyChunks=None, deleteTables=False, loggerName=None, cssDb='qservCssData',
+                 czarWmgrList=[]):
         """
         Constructor parses all arguments and prepares for execution.
 
         @param configFiles:  Sequence of the files defining all partitioning options.
-        @param czarWmgr:     WmgrClient instance for czar node.
+        @param czarWmgr:     WmgrClient instance for czar/master node.
```

Comment:
My guess here is that "czar/master" does not run czar in multi-czar mode, it is only for mysql (and cmsd)? This whole naming is a bit confusing I wish we had some distinction for what is what.

Reply:
It's the czar when when in mono mode or multinode, and the master when in multiczar. I think multinode may eventually go away as multiczar runs all of the same tests on multiple nodes, but mono is still handy as it's a lot faster to run and easier to debug. It could use a better name.